docs/examples/InputTypes.js

```diff
@@ -16,7 +16,9 @@ const inputTypeInstance = (
     </Input>
     <Input type='textarea' label='Text Area' placeholder='textarea' />
     <Input type='static' value='Static Text' />
-    <Input type='submit' value='Submit button' />
+    <ButtonInput value='Button Input' />
+    <ButtonInput type='reset' value='Reset Button' />
+    <ButtonInput type='submit' value='Submit Button' />
```

Comment:
Should we show usage of `bsStyle` here? I'm thinking that the reset would be `bsStyle='warning'` and submit would be `bsStyle='primary'`. This way folks are aware that that prop will work here.